packages/zoe/src/contractSupport/priceAuthority.js

```diff
@@ -31,7 +31,7 @@ const isGT = (amount, amountLimit) => !AmountMath.isGTE(amountLimit, amount);
 /**
  * @typedef {object} OnewayPriceAuthorityOptions
  * @property {Issuer} quoteIssuer
- * @property {ERef<Notifier<Timestamp>>} notifier
+ * @property {ERef<Notifier<unknown>>} notifier
```

Comment:
This is a relaxation of a type constraint? Any particular reason?

Reply:
It was wrong